example/pcap-analyzer/PCAP_Analyzer_README.md

```diff
@@ -0,0 +1,243 @@
+    if success(result):
+        break
+    else:
+        script = repair(script, result)
+```
+
+---
+
+## 💡 5. Prompt Template Design Philosophy
+
+| Prompt | Purpose | Core Concept |
+|---------|----------|--------------|
+| `PLANNER_SYSTEM` | Defines Planner rules | Generate Bash script, non-interactive, JSON-only output |
+| `PLANNER_USER` | Task context | Analyze `/workspace/pocket.pcap` |
+| `PLANNER_REPAIR_USER` | Auto-repair phase | Include failure logs and previous script |
+| `REPORTER_SYSTEM` | Defines Reporter behavior | Markdown report structure |
+| `REPORTER_USER` | Report template | Includes command logs and execution JSON |
+
+**Design Highlights:**
+
+- **Strict JSON output** — prevents parsing or formatting issues  
+- **Fixed Bash header:**
+  ```bash
+  #!/usr/bin/env bash
+  set -euo pipefail
+  ```
+- **Fault-tolerant logic:**  
+  Handles missing tools gracefully (`command -v tshark || true`),  
+  uses `|| true` to preserve pipeline stability.
+
+---
+
+## 🧭 6. Server Logic
+
+### Startup Sequence
+
+- Initializes the `ChatOpenAI` LLM client  
+- Creates **Planner** and **Reporter** agents  
+- Loads configurations from environment variables (e.g., `OPENAI_API_KEY`, `OPENAI_MODEL`, etc.)
+
+### `/analyze` Request Flow
+
+1. Save uploaded PCAP file (if provided).  
+2. Call `_plan_script()` → Generate initial script.  
+3. Call `_analyze_with_retries()` → Execute and auto-repair as needed.  
+4. Call `_report()` → Produce Markdown report.  
+5. Return consolidated results (`script`, `results`, `report`).
+
+---
+
+## ⚙️ 7. Configurable Parameters
+
+| Environment Variable | Description | Default |
+|----------------------|-------------|----------|
+| `OPENAI_API_KEY` | LLM API key | *Required* |
+| `OPENAI_MODEL` | Model name | `Qwen/QwQ-32B` |
+| `OPENAI_API_BASE` | Model API endpoint | `https://api.siliconflow.cn/v1` |
+| `SANDBOX_NAMESPACE` | Sandbox namespace | `default` |
+| `SANDBOX_CPU` / `SANDBOX_MEMORY` | Resource allocation | `200m` / `256Mi` |
+| `SANDBOX_WARMUP_SEC` | Sandbox warm-up time (seconds) | `20` |
+| `PLANNER_MAX_RETRIES` | Maximum auto-repair attempts | `2` |
+| `DEBUG_SAVE_DIR` | Debug artifact output directory | `./debug_artifacts` |
+
+---
+
+## 🧩 Summary
+
+**PCAP Analyzer** provides a complete **AI-driven PCAP analysis workflow**, integrating:  
+- Autonomous script generation  
+- Sandboxed execution  
+- Intelligent repair and retry cycles  
+- Automated Markdown report generation  
+
+This creates a fully automated, fault-tolerant, and self-healing forensic analysis service accessible via a simple REST API.
+
+---
+
+### 🧱 Tech Stack
+
+- **Backend:** FastAPI, Uvicorn  
+- **AI Runtime:** LangChain + LangGraph + ChatOpenAI  
+- **Execution Environment:** Custom Sandbox (agentcube)  
+- **Language:** Python 3.11+
+
+---
+
+### 🧰 Example Run
+
+```bash
+curl -X POST "http://127.0.0.1:8000/analyze"   -H "Content-Type: application/x-www-form-urlencoded"   -d "pcap_file=@./samples/pocket.pcap""
```

Comment:
Since you donot provide the guide on how to deploy this agent. But above you provide deployment yamls. So 127.0.0.1:8000 should not work